src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -221,6 +225,9 @@ public Measure createDraft(
     measureDraft.setCreatedAt(now);
     measureDraft.setLastModifiedAt(now);
     measureDraft.setCreatedBy(username);
+    if (!CollectionUtils.isEmpty(measure.getTestCases())) {
```

Comment:
I think just for QICore measures is fine. we can ignore QDM measures

Reply:
@adongare ok thanks will do.